### PR TITLE
ci: Use stable branch for packaging and osbuilder

### PR DESCRIPTION
**Pull request by jcvenegas**
osbuilder and packaging are branched now.

Fixes: #1990

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: .ci/resolve-kata-dependencies.sh [hunk 1]
```diff
@@ -94,13 +102,8 @@ clone_repos() {
 			# And show what we rebased on top of to aid debugging
 			git log --oneline "origin/${branch}~1..HEAD"
 		else
-			# Packaging repo only has master branch, so we
-			# cannot checkout to a different branch.
-			if  [ "${repo}" != "${packaging_repo}" ]
-			then
 				echo "Checking out to ${branch}"
```
Comment (chavafg) [inline .ci/resolve-kata-dependencies.sh]: remove one tab from indentation

Comment (jcvenegas) [reply .ci/resolve-kata-dependencies.sh]: done

File: .ci/resolve-kata-dependencies.sh [hunk 2]
```diff
@@ -61,7 +62,14 @@ apply_depends_on() {
 }
 
 clone_repos() {
-	local kata_repos=( "${agent_repo}" "${proxy_repo}" "${runtime_repo}" "${shim_repo}" "${tests_repo}" "${packaging_repo}" )
+	local kata_repos=(
+	"${agent_repo}"
```
Comment (jodh-intel) [inline .ci/resolve-kata-dependencies.sh]: Nice sorted list! :smile:

**Pull request merged**